### ISSUE TITLE
Gallery Bug When Editing HD and 4k Videos

### ISSUE THREAD

**Issue by GithubContributer** [SimpleMobileTools/Simple-Gallery#2237]
HI, thanks for all your hard work. However, there is a bug in your gallery app when trying to edit HD or 4k videos. It won't let me edit them in the folder section but will if I move them to the recycle bin. Strange but understood. I don't want to be a bother but it is a bug and I'm in now ways a moron in tech but not shabby either. I'm running efoundation rom I flashed myself and it's understood if the bug could be because of the rom I'm running. I've seen this happen with certain apps like open source Brave on this rom but not proprietary rom. Please continue your hard work as I've seen bugs in proprietary apps too.

**Comment by GithubContributer** [comment]
I'm running your latest release of your gallery app and am running efoundation rom (e-0.18-q-20210828132307). The files are on an SD card. By the way I purchased all your apps on googleplay to support you even though they're free on github!

**Comment by GithubContributer** [comment]
Yes it works. However, I played further with the editing in your app. It opens pictures and any video to edit in the recycle bin but does not save them. It gives no error too. However please also keep in mind which I wasn't going to mention that your gallery app rotates pictures taken in the folder section but does not in the recycle bin. I've tested proprietary apps without this problem. Outstanding work considering your not psycho stalking by your privacy policy to remove bugs. It's all hard hours put in.

**Comment by GithubContributor832** [comment]
Looking forward to this fix. Hope it's on the roadmap and near a fix. Currently the app crashes sometimes when trying to edit HD/4K videos on SD card via folder with no opening of the editor ever. Recycle bin is smooth.

**Comment by tibbi** [comment]
retry with 6.23.0, the editor has been updated

**Comment by GithubContributor832** [comment]
Still the same issue with 6.23.1.
